src/UI/Factory.php

```diff
@@ -336,4 +337,27 @@ public function panel();
 	 **/
 	public function modal();
 
+	/**
+	 * ---
+	 * description:
+	 *   purpose: >
+	 *     Popovers are used to display secondary information to a component.
+	 *   composition: >
+	 *     Popovers consist of a short title and text.
+	 *   effect: >
+	 *     Popovers are typically shown by clicking or hovering over a component. The position
+	 *     of the Popover is calculated automatically be default. However, it is possible to
+	 *     specify if the popover appears at the top, right, bottom or left of the triggerer component.
+	 * context: >
```

Comment:
Add rivals here. At the moment especially Modals come to my mind. Later tooltips will need to be added here as well. E.g. "Modals: Modals are obtrusive and hide all other content while Popovers do not prevent interaction with other parts of the context."